### ISSUE TITLE
💡 Integrate way-displays

### ISSUE THREAD

**Issue by xeruf** [manjaro-sway/manjaro-sway#484]
### Summary

[way-displays](https://github.com/alex-courtis/way-displays) is an awesome tool for auto-arranging multi-monitor setups
I previously took the effort of manually positioning all monitors in my config, but through that it becomes a breeze, and handles scaling as well.

### Drawbacks

- Sometimes monitors end up on the opposite side you intended, but that is no better with the default settings
- Scaling at odd values may make monitor less readable than no scaling at all (e.g. 1.125 rather than 1): https://github.com/alex-courtis/way-displays/issues/40
- It does not yet support non-straight setups (e.g. one monitor above and one on the side), see https://github.com/alex-courtis/way-displays/issues/79

Either way, you are better off than without it

### Implementation idea

Install and run it by default according to the README

### Will you be able to contribute a PR for this?

- [ ] Yes I will be able to get that feature a *lot* quicker.

**Comment by boredland** [comment]
We currently have `wlay` installed for a similar purpose. What makes way-displays better?

**Comment by xeruf** [comment]
ah this is also nice, did not know about it

way-displays is focused on auto-arrangement

**Comment by boredland** [comment]
this seems not to replace wlay, but kanshi. we probably should do that then!

**Comment by boredland** [comment]
added way-displays to out packages. we should create a pr replacing kanshi with it soon-ish.

**Comment by xeruf** [comment]
Now that I checked out kanshi, I am not so sure way-displays is the superior option anymore actually, I just wasn't aware of either until recently. Why do you think we should switch?

**Comment by boredland** [comment]
at a first glance, I like way-displays a bit more, we should do an in-depth comparison at least.

**Comment by boredland** [comment]
just tried this for my personal setup again and it just worked like a breeze. will integrate it.

**Comment by boredland** [comment]
@all-contributors please add @xeruf for having a great idea

**Comment by olivercoad** [comment]
way-displays prevents any other method of configuring displays from working, without any explaination or error messages.
This was quite confusing and difficult to troubleshoot until by some miracle I found this issue and found out about way-displays. (It wasn't till _after_ finding this issue that I did a search for "way-displays" and found the section in `SUPPORT.md`) Uninstalling it fixed my monitor config.

I recently installed manjaro-sway on a new setup and copied my user config accross but my screens were still positioned the wrong way around.

```
output DP-1 pos 0 0
output HDMI-1-1 pos 1680 0
```

Despite the config, the screens were still in the wrong order. I tried using `wdisplays`, but it would just reset to the default position immediately upon pressing apply. As another toubleshooting step, I tried `swaymsg output DP-1 transform 180` just to see if it would do anything, and it would flicker rotated 180 for a split second and immediately reset. The same thing happened using `wlr-randr`. Looking at the sway logs didn't give much insight into what was going on either, nor did googling things like "wayland screen configuration resets immediately".

I just wanted to report that as a user that's used manjaro-i3 and manjaro-sway for years, it was very confusing that none of the normal methods of configuring monitors I've used was working, with no error messages or anything. Since way-displays takes over configuring displays and prevents any other method from working, I think it should be opt-in only. Or at least a helpful error message or notification as a hint if it detects someone trying to configure displays another way.
